Dev/Homebrew/package_manager.7h.py

```diff
@@ -186,13 +193,117 @@ def update_all_cli(self):
         return
 
 
+class Pip2(Pip):
+
+    cli = '/usr/local/bin/pip2'
+
+
+class Pip3(Pip):
+
+    cli = '/usr/local/bin/pip3'
+
+
+class NPM(PackageManager):
+
+    cli = '/usr/local/bin/npm'
+
+    @property
+    def name(self):
+        return "npm"
+
+    def sync(self):
+        output = self.run(self.cli, '-g', '--progress=false', '--json',
+                          'outdated')
+        for package, values in json.loads(output).iteritems():
+            self.updates.append({
```

Comment:
Hopefully you fixed that :)  I don't have any linked packages (nor have I ever) so yeah, that's a new one for me :)